radicle-node/src/service/message.rs

```diff
@@ -56,6 +56,19 @@ impl From<net::SocketAddr> for Address {
     }
 }
 
+impl Address {
+    pub fn to_socket_addr(&self) -> net::SocketAddr {
```

Comment:
suggestion: `impl From<Address> for net::SocketAddr`

Reply:
Yeah will consider that; I think we might need an extra param in there, let's see.